libcaf_core/caf/typed_event_based_actor.test.cpp

```diff
@@ -168,12 +168,11 @@ string_actor::behavior_type string_reverter() {
 
 // uses `return delegate(...)`
 string_actor::behavior_type string_delegator(string_actor::pointer self,
-                                             string_actor master, bool leaf) {
-  auto next = leaf ? self->spawn(string_delegator, master, false) : master;
-  self->link_to(next);
+                                             string_actor master) {
```

Comment:
The handle is just the next actor in the chain, so let's name it accordingly.

```suggestion
                                             string_actor next) {
```